sdk/cognitiveservices/cognitiveservices-luis-authoring/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "LUISAuthoringClient Library with typescript type definitions for node.js and browser.",
   "version": "4.0.0-preview.3",
```

Comment:
The package version should have been incremented. Can you pass the package version to the autorest command when generating the code? That will take care of updating the version here and in the clientcontext file